pkg/datapath/loader/loader.go

```diff
@@ -352,17 +352,30 @@ func (l *Loader) reloadDatapath(ctx context.Context, ep datapath.Endpoint, dirs
 
 	if ip := ep.IPv4Address(); ip.IsSet() {
 		if ep.RequireEndpointRoute() {
-			upsertEndpointRoute(ep, *ip.IPNet(32))
+
+			if err := upsertEndpointRoute(ep, *ip.IPNet(32)); err != nil {
+				log.WithError(err).Error("Failed to upsert route")
```

Comment:
I think we should use the `scopedLog` here to report the endpoint.

Reply:
What should be the fields under logrus ?

Reply:
No new field needed. See previous uses of the `scopedLog` variable above in the same file.